src/functions/notifications.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+// @use in config builder
+function reportMethodUsedAbovePHP80(string $calledMethod, string $recommendedMethod): void
```

Comment:
this needs to be registered in scoper.php

https://github.com/rectorphp/rector-src/blob/61a419fe248538b2559ef1df804702e1d4392dee/scoper.php#L50